### PR TITLE
chore(docs): Highlighted `Escape` suggestion in demo for "hidden treasures". (fixes #629)

### LINKED ISSUE #629: Demo: in "hidden treasures" there is no way to deactivate the trap
In the [live demo](https://focus-trap.github.io/focus-trap/) of "hidden treasures" there is no way to deactivate the focus trap.

One could reload the page to be again free to click and tab around.

But having a 'deactivate trap' button inside of the trap would be better.

### PR DESCRIPTION
**Pull request by DaviDevMod**
As stated in the issue #629, a "deactivate trap" button is needed in the demo for "hidden treasures".

Currently this demo has a button that does "nothing" and a button that does "nothing again".  
In order to cause the least amount of changes in the source code, I turned the "nothing again" button into a "deactivate trap" button.

I did run `yarn demo-bundle` as suggested in the PR Checklist, and also references the issue solved.  
I think the other points are not applicable.

I apologise in advance if I missed to follow some good practice; this is the first time I open a PR to modify source code.

<details>
<summary>PR Checklist</summary>
<br/>

__Please leave this checklist in your PR.__

- Source changes maintain stated browser compatibility.
- Includes updated docs demo bundle if source/docs code was changed (run `yarn demo-bundle` in your branch and include the `/docs/demo-bundle.js` file that gets generated in your PR).
- Issue being fixed is referenced.
- Unit test coverage added/updated.
- E2E (i.e. demos) test coverage added/updated.
  - ⚠️ Non-covered demos (look for `IS_CYPRESS_ENV === ''` [here](https://github.com/focus-trap/focus-trap/blob/master/docs/js/index.js), as well as `in-open-shadow-dom.js` and `negative-tabindex.js` that can't be fully tested in Cypress) __manually__ verified.
- Typings added/updated.
- Changes do not break SSR:
  - Careful to test `typeof document/window !== 'undefined'` before using it in code that gets executed on load.
- README updated (API changes, instructions, etc.).
- Changes to dependencies explained.
- Changeset added (run `yarn changeset` locally to add one, and follow the prompts).
  - EXCEPTION: A Changeset is not required if the change does not affect any of the source files that produce the package bundle. For example, tooling changes, test updates, or a new dev-only dependency to run tests more efficiently should not have a Changeset since it will not affect package consumers.

</details>

### CODE REVIEW

**Comment by stefcameron** [comment]
> I did run `yarn demo-bundle` as suggested in the PR Checklist, and also references the issue solved. I thing the other points are not applicable.
> 
> I apologise in advance if I missed to follow some good practice; this is the first time I open a PR to modify source code.
>

Thanks for the proactive PR for the issue you filed, I appreciate it. Looks like you followed the checklist well! 🎉 

I have a question for you back on the issue, though...

**Comment by DaviDevMod** [comment]
I should have definitely run the tests. I didn't because at the moment I am quite ignorant in matter of tests.

As I said back in the issue, I am sorry for the inconvenience. I close the PR.

**Comment by DaviDevMod** [comment]
I put "Escape" in bold and its suggestion on a new line. I think that this should do the job.

I didn't rebase to include the commits that in the meantime have been added to your master, because I guess that I would have lost the ability to modify this PR.

This time I did run the tests :sweat_smile:

File: docs/index.html [hunk 1]
```diff
@@ -235,7 +235,7 @@ <h2 id="ifc-heading">initially focused container</h2>
     <div id="demo-ht">
       <h2 id="ht-heading">hidden treasures</h2>
       <p>
-        Focusable nodes are initially hidden and then revealed within the trap. Use <kbd>Escape</kbd> to exit.
+        Focusable nodes are initially hidden and then revealed within the trap.<br/>Use <b><kbd>Escape</kbd></b> to exit.
       </p>
```
[the remark this answers is not in the document]
Comment (DaviDevMod) [reply docs/index.html:239]: Of course!

File: .changeset/fair-adults-rhyme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'focus-trap': patch
+---
+
+Highlighted the possibility to use `Escape` to leave the trap, in the demo for "hidden treasures".
```
[the remark this answers is not in the document]
Comment (DaviDevMod) [reply .changeset/fair-adults-rhyme.md:5]: No problem, I'm going to push.

**Review by stefcameron — APPROVED** [review]
Awesome!

**Comment by stefcameron** [comment]
@all-contributors add @DaviDevMod for docs

**Comment by stefcameron** [comment]
@DaviDevMod Thank you for your work here. I appreciate it!

**Comment by DaviDevMod** [comment]
@stefcameron Thank you for having guided me through this PR. I am glad we made something good out of the initial misunderstanding.

**Pull request merged**